src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    list-style: none;
+    margin: 0 10px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    @include font(12px, 500, 60px);
+
+    display: block;
+    text-decoration: none;
+    align-content: center;
```

Comment:
The property 'align-content' is not applicable to 'display: block'. Consider using 'text-align' or another relevant property for aligning text within a block element.